Improve the message on contribution PR to an old pack versions (#2935)
demisto/demisto-sdk: CHANGELOG.md

# Changelog

## Unreleased

> * Improved the warning message that that is displayed for Contribution PRs that update outdated packs.

Reviewer: ```suggestion
* Improved the warning message displayed for Contribution PRs editing outdated code.
```
